web/client/plugins/CRSSelector.jsx

```diff
@@ -40,7 +41,9 @@ class Selector extends React.Component {
         typeInput: PropTypes.func,
         enabled: PropTypes.bool,
         currentBackground: PropTypes.object,
-        onError: PropTypes.func
+        onError: PropTypes.func,
+        AllowedUsers: PropTypes.array,
+        currentRole: PropTypes.string
     };
     static defaultProps = {
         availableCRS: CoordinatesUtils.getAvailableCRS(),
```

Comment:
set `allowedRoles: "ALL"` by default, this may help to make the crs selector usable without customizations.